src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/EHInfo.cs

```diff
@@ -25,7 +25,6 @@ public enum CorExceptionFlag
         COR_ILEXCEPTION_CLAUSE_FILTER = 0x0001,         // If this bit is on, then this EH entry is for a filter
         COR_ILEXCEPTION_CLAUSE_FINALLY = 0x0002,        // This clause is a finally clause
         COR_ILEXCEPTION_CLAUSE_FAULT = 0x0004,          // Fault clause (finally that is called on exception only)
-        COR_ILEXCEPTION_CLAUSE_DUPLICATED = 0x0008,     // duplicated clause. This clause was duplicated to a funclet which was pulled out of line
```

Comment:
ILCompiler.Reflection.ReadyToRun is expected to be able to dump older versions. The changes in this file should be reverted.